src/main/java/com/hypherionmc/modpublisher/tasks/GithubUploadTask.java

```diff
@@ -96,6 +96,11 @@ public void upload() throws Exception {
 
         // Existing release was not found, so we create a new one
         if (ghRelease == null) {
+             if (!extension.getCreateGithubRelease().get()) {
+                 project.getLogger().warn("Create GitHub Release is disabled and Github Release with tag " + tag + " does not exist");
```

Comment:
> since this would technically be an error

Had the same thought process but came to the opposite conclusion :joy: 

IMO this would be configurable too, since some users will want to fail while others would want to "attempt" to publish, unless exists/missing

Reply:
That's fair. Might as well just leave it at a warning lol